dist/angular-ui-tree.js

```diff
@@ -628,11 +628,6 @@
               firstMoving = true;
               dragInfo = UiTreeHelper.dragInfo(scope);
 
-              // Fire dragStart callback
-              scope.$apply(function () {
-                scope.$treeScope.$callbacks.dragStart(dragInfo.eventArgs(elements, pos));
```

Comment:
You're right. Can you bring the PR up to date with master so I can merge this?
Thanks for your work! :cake: